<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>811</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Ajouter Reservation</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>541</width>
        <height>261</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>Formulaire d'Ajout </string>
      </property>
      <widget class="QLineEdit" name="le_code">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>40</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_cin">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>80</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>120</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_25">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Code_Res</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_26">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>CIN</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_27">
       <property name="geometry">
        <rect>
         <x>46</x>
         <y>120</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Date Res</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_28">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>200</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Service</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_39">
       <property name="geometry">
        <rect>
         <x>46</x>
         <y>160</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Heure Res</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_heur">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>160</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>200</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>a</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>b</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>c</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>d</string>
        </property>
       </item>
      </widget>
      <widget class="QDateEdit" name="le_date">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>120</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Afficher  Reservation</string>
     </attribute>
     <widget class="QTableView" name="tableView_Res">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>511</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="supp">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>170</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="xl">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>210</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Exel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="search">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>40</y>
        <width>391</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="placeholderText">
       <string>rechercher...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tri">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Trier Par Date</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pdf">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>250</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>Modifier Reservation</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>501</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>Formulaire modifier</string>
      </property>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Code_Res</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushmod_2">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>50</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_code_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>60</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_heur_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>140</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QLabel" name="label_29">
       <property name="geometry">
        <rect>
         <x>54</x>
         <y>180</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Service</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_30">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>100</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Date Res</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_40">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>140</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Heure Res</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>180</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>a</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>b</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>c</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>d</string>
        </property>
       </item>
      </widget>
      <widget class="QDateEdit" name="date_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>100</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Historique</string>
     </attribute>
     <widget class="QTextBrowser" name="Histo">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>721</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>942</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
